nx.json

```diff
@@ -1,40 +1,24 @@
 {
   "implicitDependencies": {
-    "package.json": {
-      "dependencies": "*",
-      "devDependencies": "*"
-    },
+    "package.json": { "dependencies": "*", "devDependencies": "*" },
     ".eslintrc.json": "*"
   },
-  "affected": {
-    "defaultBase": "master"
-  },
+  "affected": { "defaultBase": "origin/main" },
```

Comment:
why `origin/main` and not just `main`?

Reply:
Because this way probably running affected stuff won't fail in github actions, and we don't have to check out all the git history later (fetch-depth: 0).